src/plot_api/plot_api.js

```diff
@@ -1708,6 +1708,7 @@ function addAxRangeSequence(seq, rangesAltered) {
         subroutines.doTicksRelayout;
 
     seq.push(
+        subroutines.doAutoRangeAndConstraints,
```

Comment:
Adding `doAutoRangeAndConstraints` to the 'axrange' update sequence does work against some of the work done in https://github.com/plotly/plotly.js/pull/2628, but only by 1ms in the most extremes case (a 50x50 splom) when there are no axis constraints.